src/pages/program/[slug]/[profileRoute]/dashboard.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Fragment } from "react";
+import { Card, Text } from "../../../../components/atomic";
+import {
+  ProfileType,
+  useGetProfilesQuery,
+} from "../../../../generated/graphql";
+import { AuthorizationLevel, useCurrentProgram } from "../../../../hooks";
+import AuthorizationWrapper from "../../../../layouts/AuthorizationWrapper";
+import ChooseTabLayout from "../../../../layouts/ChooseTabLayout";
+import PageContainer from "../../../../layouts/PageContainer";
+import Page from "../../../../types/Page";
+
+const DashboardPage: Page = () => {
+  const { currentProgram } = useCurrentProgram();
+  const { programId } = currentProgram || {
+    programId: ""
+  };
+
+  const mentors = useGetProfilesQuery({
```

Comment:
nice!

Reply:
thanks! code in settings.tsx was very helpful B)